bitnami/postgresql/templates/secrets.yaml

```diff
@@ -1,8 +1,14 @@
 {{- if (include "postgresql.createSecret" .) }}
+{{- $requiredPasswords := list "postgresqlPassword" -}}
+{{- $secretName :=  include "postgresql.fullname" . -}}
+
+{{- $getpostgresqlPassword := include "common.notetxt.secret.getvalue" (dict "varname" "POSTGRESQL_PASSWORD" "secret" $secretName "field" "postgresql-password" "context" . ) -}}
+{{- $getPasswords := list $getpostgresqlPassword -}}
+
```

Comment:
I won't include the extra blank lines. Up to you

Reply:
N/A